### PR TITLE
Sylius 1.11 support

### PR DESCRIPTION
**Pull request by marekrzytki**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
             matrix:
                 php: [7.4, 8.0]
                 symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.10.0]
+                sylius: [~1.8.0, ~1.10.0, ~1.11.0]
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml]: Can you add 1.9?

File: tests/Application/.env.dist [hunk 2]
```diff
@@ -15,6 +15,20 @@ APP_SECRET=EDITME
...
+###> lexik/jwt-authentication-bundle ###
+JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
+JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+JWT_PASSPHRASE=acme_plugin_development
+###< lexik/jwt-authentication-bundle ###
+
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+#MESSENGER_TRANSPORT_DSN=doctrine://default
```
Comment (jakubtobiasz) [inline tests/Application/.env.dist]:
```suggestion
# MESSENGER_TRANSPORT_DSN=doctrine://default
```

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]

File: composer.json [hunk 3]
```diff
@@ -39,12 +39,26 @@
...
+            "phpstan/extension-installer": true
+        }
+    },
+    "conflict": {
+        "doctrine/dbal": "^3.0",
+        "symfony/symfony": "4.1.8",
+        "symfony/browser-kit": "4.1.8",
+        "symfony/dom-crawler": "4.1.8",
+        "symfony/routing": "4.1.8",
+        "symfony/doctrine-bridge": "4.4.16"
```
Comment (senghe) [inline composer.json:60]: I don't get why we have those changes

Comment (marekrzytki) [reply composer.json:60]: It resolves problems with Sylius 1.9 build

**Review by senghe — APPROVED** [review]

**Pull request merged**